doc/user_guide/dynamo0p3.rst

```diff
@@ -892,7 +892,13 @@ vector variables are:
 
 * ``W2trace`` is the space of scalar functions defined only on cell faces,
   resulting from taking the trace of a ``W2`` space. DoFs are shared between
-  faces, hence making this space fully continuous.
+  faces, hence making this space fully continuous;
+
+* ``Wchi`` is the space of scalar functions used to store coordinates in
+  LFRic, fully discontinuous except for the coordinate order ``0`` when it
+  becomes the ``W0`` space (i.e. fully continuous). Since LFRic neither
```

Comment:
As agreed, I created issue #749 to prevent write accesses and halo exchanges for `Wchi` space. I added the appropriate TODO references in `dynamo0p3.py`.